tests/test_reports.py

```diff
@@ -31,234 +31,139 @@ def test_get_application_statuses(client):
-            "questions": [
-                {
-                    "question": "Address",
-                    "fields": [
-                        {
-                            "key": "yEmHpp",
-                            "title": "Address",
-                            "type": "text",
-                            "answer": "BBC, BA2 1AA",
-                        },
-                    ],
-                },
-            ],
-            "metadata": {
-                "application_id": app_two.id,
-                "form_name": "project-information",
-                "is_summary_page_submit": False,
-            },
-        },
-    ]
-
-    for section in sections_put_app_one:
-        client.put(
-            "/applications/forms",
-            json=section,
-            follow_redirects=True,
-        )
-    for section in sections_put_app_two:
-        client.put(
-            "/applications/forms",
-            json=section,
-            follow_redirects=True,
-        )
-    app_one.status = Status.IN_PROGRESS
-    app_two.status = Status.IN_PROGRESS
-    apps[2].status = Status.IN_PROGRESS
+    assert 200 == response.status_code
+    lines = response.data.splitlines()
+    assert 2 == len(lines)
+    assert (
+        "eoi_reference,organisation_name,organisation_type,asset_type,"
+        + "geography,capital,revenue"
+    ) == lines[0].decode("utf-8")
+    fields = lines[1].decode("utf-8").split(",")
+    assert "Test Org Name 1" == fields[1]
+    assert "Test Reference Number" == fields[0]
+    assert "W1A 1AA" == fields[4]
+
+
+@pytest.mark.fund_round_config(
+    {
+        "funds": [
+            {"rounds": [{"applications": [{**user_lang}, {**user_lang}]}]},
+        ]
+    }
+)
+def test_get_applications_report_query_param(
+    client, seed_data_multiple_funds_rounds
+):
 
     response = client.get(
-        "/applications/reporting/key_application_metrics?status=IN_PROGRESS",
+        "/applications/reporting/key_application_metrics?status=IN_PROGRESS&"
+        + f"fund_id={seed_data_multiple_funds_rounds[0].fund_id}&round_id="
+        + f"{seed_data_multiple_funds_rounds[0].round_ids[0].round_id}",
         follow_redirects=True,
     )
-    report_reponse = response.data.decode("utf-8")
-    assert "organisation_name" in report_reponse
-    assert "Test Organisation Name" in report_reponse
-    assert "Test Organisation Name 2" in report_reponse
-    assert "Test Reference Number" in report_reponse
-    assert "W1A 1AA" in report_reponse
-    assert "BA2 1AA" in report_reponse
+    lines = response.data.splitlines()
+    assert 3 == len(lines)
+    assert (
+        "eoi_reference,organisation_name,organisation_type,asset_type,"
+        + "geography,capital,revenue"
+    ) == lines[0].decode("utf-8")
+    for line in lines[1:]:
+        fields = line.decode("utf-8").split(",")
+        assert fields[1].startswith("Test Org Name ")
+        assert "Test Reference Number" == fields[0]
+        assert "W1A 1AA" == fields[4]
```

Comment:
I think some of these assertions are backwards, the constant or "expected" value should be on the right, not a big deal just pytest will report backwards.

Small potential refactoring, I like explicit destructing as it gives the viewer quick reference as to how many elements can be in a sequence (more of a suggestion/subjective thing, feel free to ignore):

```suggestion
    raw_lines = response.data.splitlines()
    assert len(raw_lines) == 3

    line1, line2, line3 = [
        line.decode("utf-8") for line in response.data.splitlines()
    ]
    assert line1 == (
       "eoi_reference,organisation_name,organisation_type,asset_type,"
       "geography,capital,revenue"
    )
    for line in line2, line3:
        field1, field2, _, _, field5, _, _ = line.split(",")
        assert field1 == "Test Reference Number"
        assert field2.startswith("Test Org Name ")
        assert field5 == "W1A 1AA"
```